dimecoin-qt.pro

```diff
@@ -30,40 +32,40 @@ greaterThan(QT_MAJOR_VERSION, 4) {
 #    BOOST_INCLUDE_PATH, BOOST_LIB_PATH, BDB_INCLUDE_PATH,
 #    BDB_LIB_PATH, OPENSSL_INCLUDE_PATH and OPENSSL_LIB_PATH respectively
 
-###############################################
-# Uncomment to build on Ubuntu
+##############################################
+# Wherever you compile the dependencies, 
+# specify the path here (linux, windows)
 # ############################################
-# BOOST_LIB_PATH=/usr/local/BerkeleyDB.5.0/lib
-# BDB_INCLUDE_PATH=/usr/local/BerkeleyDB.5.0/include
+DEPS_PATH = /home/alex/deps
```

Comment:
This doesn't seem right, appears specific to your file system. Please double check.

Reply:
@CryptoAxl said -

In this Line there is a specific comment indicating to change to other dep path were other dependencies are compiled. I can put the usual use/local path or c:/deps it's the same result: it will need to be set by anyone who will want to compile on his machine. Still need to upload both linux and Windows guide to compile statically

Reply:
Okay, that seems fine; so long as you annotate it in the guides.